frontend/src/components/apollo-client.ts

```diff
@@ -1,7 +1,7 @@
 import { ApolloClient, InMemoryCache } from "@apollo/client";
```

Comment:
apollo-client file shouldn't be in components folder — find a different spot for it 😉

Reply:
Done!